loads/src/main/java/org/mousephenotype/cda/loads/LoaderUtils.java

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+ * Copyright © 2015 EMBL - European Bioinformatics Institute
+ * <p/>
+ * Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the
+ * License.
+ ******************************************************************************/
+
+package org.mousephenotype.cda.loads;
+
+/**
+ * Created by mrelac on 20/07/16.
+ */
+public class LoaderUtils {
+
+
+    /**
+     * Maps external input names to CDA known names for project, organisation, ontology term, etc.
+     *
+     * Key terms in this array (column 0) should be entered in lowercase.
+     */
+    private final String[][] mappedTerms = new String[][] {
```

Comment:
use a Map<String, String> instead of double array, then the translatTerm method is:

return mappedTerms.containsKey(term) ? mappedTerms.get(term) : term;